js_modules/dagster-ui/packages/ui-core/src/assets/useWipeAssets.tsx

```diff
@@ -30,6 +44,31 @@ export function useWipeAssets({
 
   const didCancel = useRef(false);
 
+  if (isWiping && backgroundWipeStatusError) {
+    setFailedCount(1);
+    onComplete?.();
+    setIsWiping(false);
+  }
+
+  if (isWiping && backgroundWipeStatusData) {
+    const data = backgroundWipeStatusData.backgroundAssetWipeStatus;
+    switch (data.__typename) {
+      case 'BackgroundAssetWipeInProgress':
+        console.log('Background asset wipe in progress');
+        break;
+      case 'BackgroundAssetWipeSuccess':
+        setWipedCount(1);
+        onComplete?.();
+        setIsWiping(false);
+        break;
+      case 'BackgroundAssetWipeFailed':
+        setFailedCount(1);
+        onComplete?.();
+        setIsWiping(false);
+        break;
+    }
+  }
```

Comment:
Avoid setting state or performing side effects within the render path. These should occur in callbacks or effects.